core/src/main/java/org/opensearch/sql/expression/text/TextFunction.java

```diff
@@ -241,6 +242,18 @@ private DefaultFunctionResolver locate() {
                 TextFunction::exprLocate), INTEGER, STRING, STRING, INTEGER));
   }
 
+  /**
+   * Returns the position of the first occurrence of a substring in a string.
```

Comment:
Can you mention that it starts with position 1, rather than 0? Since 0 is a 'failure'.